tensorflow_lite_support/python/task/processor/processor_options.py

```diff
@@ -31,3 +31,33 @@ class EmbeddingOptions:
   # therefore any dimension is guaranteed to have a value in [-1.0, 1.0]. Use
   # the l2_normalize option if this is not the case.
   quantize: Optional[bool] = None
+
+
+@dataclasses.dataclass
+class ClassificationOptions:
+  """Options for classification processor."""
+  # The locale to use for display names specified through the TFLite Model
```

Comment:
please use docstring instead of comment so that it'll show up in API docs later. For example:

```python
display_names_locale: Optional[str] = None
"""Metadata, if any. Defaults to English."""
```